bom/runtime/pom.xml

```diff
@@ -2418,6 +2425,11 @@
                 <artifactId>vertx-web-common</artifactId>
                 <version>${vertx.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.vertx</groupId>
+                <artifactId>vertx-grpc</artifactId>
+                <version>${vertx.version}</version>
```

Comment:
This one is not in the Vert.x bom we include?

Reply:
It is! Good point.

Reply:
Fixed, I've also removed some other dependencies that should not have been there (for the same reason)